project.clj

```diff
@@ -25,7 +32,19 @@
                                               javax.jms/jms
                                               com.sun.jdmk/jmxtools
                                               com.sun.jmx/jmxri]]]
-  :plugins [[michaelblume/lein-marginalia "0.9.0"]]
+  :plugins [[michaelblume/lein-marginalia "0.9.0" :exclusions [org.clojure/clojurescript]]
+            [lein-figwheel "0.5.0-2" :exclusions [joda-time]]]
   :main ^:skip-aot doctopus.web
   :target-path "target/%s"
+  :cljsbuild {:builds [{:id "dev"
```

Comment:
Should probably open an issue to make a prod cljs build profile, so we don't lose track